types/migrations/20220518174701_edited_collections.sql

```diff
@@ -0,0 +1,28 @@
+-- Add migration script here
+
+-- A file for collections and chapters
+
+create extension if not exists "ltree";
+
+create table collections (
+  id autouuid primary key,
+  collection_name text not null,
+  collection_desc text,
+  slug ltree not null unique -- keep it the same type as chapter ltree
```

Comment:
Collections won't ever be nested within each other, so the slug should just be `text`